Add output checking statements (#199)
sa-tre/satre-specification: docs/source/roles.md

* - Role name
  - Role description
* - Output checker

>   - Person or people responsible disclosure control via checking of project outputs destined for egress from the TRE.

Reviewer: ```suggestion
  - People responsible for checking the disclosure risk of project outputs, before egress, as part of the disclosure control process.
```

I'm not sure it is correct to say the output checkers are responsible for disclosure control as a whole.
